### PR TITLE
[Fix] Lowers pda sounds as well as make them respect ringer off

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]

🆑
fix: Makes most pda sounds softer and if ringer or silenced will no longer make the beeps and boops
/🆑

[why]
Kev wanted to remove them!
Im not Lazy
This is legit tested by the by
Helping

### CODE REVIEW

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -445,73 +450,85 @@ GLOBAL_LIST_EMPTY(PDAs)
...
 				else if((!isnull(cartridge)) && (cartridge.access & CART_ENGINE))
 					scanmode = PDA_SCANNER_HALOGEN
-				playsound(src, 'sound/machines/terminal_select.ogg', 50, 1)
+				if (!silent)
+					playsound(src, 'sound/machines/terminal_select.ogg', 15, 1)
 
 			if("Honk")
 				if ( !(last_noise && world.time < last_noise + 20) )
-					playsound(src, 'sound/items/bikehorn.ogg', 50, 1)
+					playsound(src, 'sound/items/bikehorn.ogg', 15, 1)
```
Comment (Poojawa) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded change

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -445,73 +450,85 @@ GLOBAL_LIST_EMPTY(PDAs)
...
 					last_noise = world.time
 
 			if("Trombone")
 				if ( !(last_noise && world.time < last_noise + 20) )
-					playsound(src, 'sound/misc/sadtrombone.ogg', 50, 1)
+					playsound(src, 'sound/misc/sadtrombone.ogg', 15, 1)
```
Comment (Poojawa) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded change

**Review by Poojawa — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 3]
```diff
@@ -391,7 +391,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 //BASIC FUNCTIONS===================================
 
 			if("Refresh")//Refresh, goes to the end of the proc.
-				playsound(src, 'sound/machines/terminal_select.ogg', 50, 1)
+				playsound(src, 'sound/machines/terminal_select.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: dont forget the silent checks

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 4]
```diff
@@ -696,7 +716,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 	tnote += "<i><b>&larr; From <a href='byond://?src=[REF(src)];choice=Message;target=[REF(signal.source)]'>[signal.data["name"]]</a> ([signal.data["job"]]):</b></i><br>[signal.format_message()]<br>"
 
 	if (!silent)
-		playsound(src, 'sound/machines/twobeep.ogg', 50, 1)
+		playsound(src, 'sound/machines/twobeep.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded change, sound is already quiet enough

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 5]
```diff
@@ -732,10 +752,12 @@ GLOBAL_LIST_EMPTY(PDAs)
...
 	if(id)
 		remove_id()
-		playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/terminal_eject_disc.ogg', 15, 1)
 	else
 		remove_pen()
-		playsound(src, 'sound/machines/button4.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button4.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded volume decrease, sound is already quiet enough

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 6]
```diff
@@ -732,10 +752,12 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	if(id)
 		remove_id()
-		playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/terminal_eject_disc.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded change, sound is already quiet

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 7]
```diff
@@ -445,73 +450,85 @@ GLOBAL_LIST_EMPTY(PDAs)
 					cartridge.host_pda = null
 					cartridge = null
 					update_icon()
-				playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
+				if (!silent)
+					playsound(src, 'sound/machines/terminal_eject_disc.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded volume decrease, sound is already quiet

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 8]
```diff
@@ -863,7 +889,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 			to_chat(user, "<span class='notice'>You slide \the [C] into \the [src].</span>")
 			inserted_item = C
 			update_icon()
-			playsound(src, 'sound/machines/button.ogg', 50, 1)
+			if (!silent)
+				playsound(src, 'sound/machines/button.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: unneeded volume decrease, sound is already quiet

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 9]
```diff
@@ -811,7 +834,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 		if(old_id)
 			user.put_in_hands(old_id)
 		update_icon()
-		playsound(src, 'sound/machines/button.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: Sound is already quiet, volume change can be reverted. Also, this is a mechanical sound, making this mutable via a software toggle makes no logical sense

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 10]
```diff
@@ -823,21 +847,24 @@ GLOBAL_LIST_EMPTY(PDAs)
 		cartridge.host_pda = src
 		to_chat(user, "<span class='notice'>You insert [cartridge] into [src].</span>")
 		update_icon()
-		playsound(src, 'sound/machines/button.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button.ogg', 15, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: Sound is already quiet, volume change can be reverted. Also, this is a mechanical sound, making this mutable via a software toggle makes no logical sense

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 11]
```diff
@@ -732,10 +753,12 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	if(id)
 		remove_id()
-		playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: This is a mechanical sound, making this mutable via a software toggle makes no logical sense

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 12]
```diff
@@ -732,10 +753,12 @@ GLOBAL_LIST_EMPTY(PDAs)
...
 	if(id)
 		remove_id()
-		playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
 	else
 		remove_pen()
-		playsound(src, 'sound/machines/button4.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button4.ogg', 50, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: This is a mechanical sound, making this mutable via a software toggle makes no logical sense

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 13]
```diff
@@ -863,7 +890,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 			to_chat(user, "<span class='notice'>You slide \the [C] into \the [src].</span>")
 			inserted_item = C
 			update_icon()
-			playsound(src, 'sound/machines/button.ogg', 50, 1)
+			if (!silent)
+				playsound(src, 'sound/machines/button.ogg', 50, 1)
```
Comment (deathride58) [inline code/game/objects/items/devices/PDA/PDA.dm]: This is a mechanical sound, making this mutable via a software toggle makes no logical sense

**Review by deathride58 — DISMISSED** [review]

**Review by silicons — APPROVED** [review]

**Pull request merged**